fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -1706,26 +1706,38 @@ public class Config extends ConfigBase {
     /**
      * iceberg metadata cache dir
      */
-    @ConfField
+    @ConfField(mutable = true)
     public static String iceberg_metadata_cache_disk_path = StarRocksFE.STARROCKS_HOME_DIR + "/caches/iceberg";
 
     /**
-     * iceberg metadata cache total size, default 2GB
+     * iceberg metadata memory cache total size, default 512MB
      */
-    @ConfField
-    public static long iceberg_metadata_cache_capacity = 2147483648L;
+    @ConfField(mutable = true)
```

Comment:
Why mutable? you can change the iceberg cache capacity at FE running?

Reply:
up to now, it's not ok, but we may need support it on saas.